lldb-eval/parser.cc

```diff
@@ -1988,6 +1999,17 @@ ExprResult Parser::BuildCStyleCast(Type type, ExprResult rhs,
     }
     kind = CStyleCastKind::kPointer;
 
+  } else if (type.IsNullPtrType()) {
+    // Cast to nullptr type.
+    if (!rhs_type.IsNullPtrType() && !rhs->is_literal_zero()) {
+      BailOut(ErrorCode::kInvalidOperandType,
+              llvm::formatv("cannot cast from type '{0}' to type '{1}'",
```

Comment:
Is this the message `clang` would give in this context? I've tried `(std::nullptr_t)1` and got:
```
<source>:8:14: error: C-style cast from 'int' to 'std::nullptr_t' (aka 'nullptr_t') is not allowed
    auto x = (std::nullptr_t)1;
             ^~~~~~~~~~~~~~~~~
```

Reply:
That's right, I updated the message.

Also, note that `static_cast<std::nullptr_t>(1)` results with the same "C-style" message, that should be fixed when we improve C++ casts handling.